### PR TITLE
Use Roboto as default font

### LINKED ISSUE #178: Use platform-independent default font
Currently, the used default font is system-dependent such that the launcher icon is slightly different if build on different machines.

I realized this while running the screenshot tests. The current screenshots seem to be generated on macOS 11 that uses "Helvetica Neue" as default font. When I build the screenshots on Windows, the results are slightly different.

### Proposal
Use `Roboto` in case now font is provided such that the generated launcher icon is independent of the build system.

## Example

|macOS|Windows|
|---|---|
|macOS.png?raw=true)|Windows.png?raw=true)|

### PR DESCRIPTION
**Pull request by brudaswen**
Package `Roboto-Regular.ttf` into the plugin and use this font in case no explicit font is provided.

Will generate the new screenshots as a comparison later.

Closes #178

### CODE REVIEW

File: easylauncher/src/main/kotlin/com/project/starter/easylauncher/filter/DefaultFont.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.project.starter.easylauncher.filter
+
+import java.awt.Font
+
+val DEFAULT_EASYLAUNCHER_FONT: Font =
+    EasyLauncherFilter::class.java.classLoader
+        .getResourceAsStream("Roboto-Regular.ttf").use {
+            Font.createFont(Font.TRUETYPE_FONT, it)
+        }
```
Comment (mateuszkwiecinski) [inline easylauncher/src/main/kotlin/com/project/starter/easylauncher/filter/DefaultFont.kt:9]: 💭 
Knowing that's an IO operation I'd suggest to make it lazy, so it doesn't get loaded when not needed. 
Also I wouldn't like to make it public if it doesn't have to be

```suggestion
internal val DEFAULT_EASYLAUNCHER_FONT: Font = by lazy {
    EasyLauncherFilter::class.java.classLoader
        .getResourceAsStream("Roboto-Regular.ttf").use {
            Font.createFont(Font.TRUETYPE_FONT, it)
        }
}
```

**Comment by brudaswen** [comment]
I re-generated the screenshots. However, I am not able to generate `example-activity-alias/Icons_doScreenshot(release).png` and `example-activity-alias/Icons_doScreenshot(unspecified).png` since the `recordReleaseAndroidTestScreenshotTest` and `recordUnspecifiedAndroidTestScreenshotTest` tasks do not exist.

**Comment by mateuszkwiecinski** [comment]
Android Gradle Plugin creates test tasks only for single build type :/ To make mentioned tasks available you have to pass `testBuildType` argument. 

Here's the code responsible for switching selected build type: https://github.com/usefulness/easylauncher-gradle-plugin/blob/28ff5c96d03a75a831eaa6114268e7de17406aa3/sample/example-activity-alias/build.gradle#L5

So the commands would be something like:
`gradle recordUnspecifiedAndroidTestScreenshotTest -PtestBuildType=unspecified`
`gradle recordReleaseAndroidTestScreenshotTest -PtestBuildType=relese`

(and seeing one of your previous commits - I'm not using Gradle Wrapper for test projects to make sure CI will test them with different Gradle versions. If you don't have gradle installed globally you can always use wrapper from parent location `../gradlew`)

**Review by mateuszkwiecinski — APPROVED** [review]
I'm not sure what's happening here precisely, but fore some reason screenshot tests don't work on CI. They don't fail despite comparing two obviously different images 🤷 The same setup works for me locally and for example current `master` properly fails with expected output:
image

Moreover, screenshots generated in this PR are a bit different than those I generate locally from your branch 🤔 They seem have a bit different transparency:
https://github.com/usefulness/easylauncher-gradle-plugin/commit/16422174daf25f59b4864b09f9973b8e879cd886

If you want, we can merge this PR in current form and I'll keep looking into tests setup

**Comment by mateuszkwiecinski** [comment]
@brudaswen I was finally able to resolve the screenshots issue. They didn't work because I was using wrong Python version. Facebook's plugin doesn't support Python 3 :/ (also it turned out I should also pay attention to java version when re-generating screenshots)

I wasn't able to push to this branch so I cherry-picked your commit and created separate PR: https://github.com/usefulness/easylauncher-gradle-plugin/pull/184

Thank you once again for spotting the issue and fixing it 🙏

**Comment by brudaswen** [comment]
Cool. Thanks for taking care.

**Pull request closed without merging**